plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -47,4 +47,20 @@ public void downloadOldSchema() throws Exception {
         );
         writer.close();
     }
+
+
+    @Test
+    public void validateSchema() {
+
+    }
+
+    @Test
+    public void checkRootConfigurators() {
+
+    }
+
+    @Test
+    public void checkInitialTemplate() {
```

Comment:
@casz @oleg-nenashev Thought of testing the schema for the inital template as well.Should I store a prewritten and correct version of it in the resources folder and then validate ?